services/core-api/app/api/exports/static_content/resources/core_static_content_resource.py

```diff
@@ -94,7 +95,8 @@ class StaticContentResource(Resource):
     )
     @requires_role_view_all
     def get(self):
-        content_json = cache.get(STATIC_CONTENT_KEY)
+        # content_json = cache.get(STATIC_CONTENT_KEY)
+        content_json = {}
```

Comment:
Was this accidentally left in, or was it intended?

Reply:
that was a mistake, good catch - updated.